app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -220,29 +221,26 @@ class CiYaml {
   /// Pick targets out of the given [targets] list that should be run for the
   /// branch that's being tested.
   List<Target> _selectTargetsForBranch(List<Target> targets) {
-    final isReleaseBranch = branch.contains(RegExp('^flutter-'));
-
-    if (isReleaseBranch) {
-      // For release branches we don't want to run release targets or bringup
-      // targets because they are built outside of Cocoon. This applies in both
-      // fusion and non-fusion repos.
+    if (!isFusion) {
+      // Non-flutter/flutter repos run all targets in post-submit.
+      return targets;
+    } else if (isReleaseCandidateBranch(branchName: branch)) {
+      // For release branches:
+      // bringup: true         --> Always omitted.
+      // release_build: "true" --> Built by Linux flutter_release_builder.
       return [
         ...targets.where(
           (target) => !target.isReleaseBuild && !target.isBringup,
         ),
       ];
+    } else if (branch.contains('experimental')) {
+      // For experimental branches:
+      // Run all targets in post-submit.
```

Comment:
Might be good to note that this will produce binaries and upload them to gcp buckets.